css/style.css

```diff
@@ -1,9 +1,203 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
 body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  display: flex;
+  flex-direction: column;
+  padding-left: 5%;
+  padding-right: 5%;
+  margin-left: 5%;
+  margin-right: 5%;
+}
+
+header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  margin-bottom: 0%;
+  padding-bottom: 0%;
+}
+
+.icodiv {
+  width: 20%;
+  margin-left: 1%;
+  padding-left: 1%;
+  margin-bottom: 0%;
+}
+
+.icoimage {
+  width: 15%;
+  height: 15%;
+}
+
+.navdiv {
+  width: 50%;
+  margin-bottom: 0%;
+}
+
+.unorderlist {
+  display: flex;
+  flex-direction: row;
+  padding: 5%;
+  justify-content: space-around;
+  list-style-type: none;
+  font-size: 20px;
+}
+
+a {
+  color: rgb(128, 118, 118);
+  text-decoration: none;
+}
+
+.meetkar {
+  font-weight: 600;
+}
+
+a:hover {
+  color: red;
+}
+
+main {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 0%;
+  margin-bottom: 10%;
+}
+
+.maindivimage {
+  position: relative;
+  width: 120%;
+  text-align: center;
+  color: white;
+}
+
+.mainimag {
+  max-width: 100%;
+  height: 80%;
```

Comment:
When sizing an `img` tag, never give it a width AND a height as this will cause the image to distort. It will try and stretch the image inside the foxed box that you have assigned it and make it look squashed or elongated. 
It's best if you just give the image `width: 100%` for example, and let the height set naturally.

Reply:
In this particular case however, I would not use an `img` tag for this section. 
There is a great way to add a background image in CSS, using the `background-image` property. You could add this directly to you `maindivimage` class.